src/js/modules/infragistics.ui.editors.js

```diff
@@ -1592,7 +1593,22 @@
 				$(".selector").%%WidgetName%%("option", "suppressNotifications", true);
 			```
 			*/
-			suppressNotifications: false
+			suppressNotifications: false,
+			/* type="bool" Suppress on-screen keyboard from showing when the dropdown button is clicked and the list is opened.

```

Comment:
If the option name is set for consistency, can we at least make this extra clear in the description?
- Option "prevents initial **or** removes focus when the drop button is clicked". Yes the use case is for mobile on-screen ones, but the actual functionality is generic. Also to avoid confusion that this is not a "mobile-readonly"
- There should be a `Note:` clearly stating that this option does no auto-detection whatsoever, meaning it's left to the application to figure out dynamically when to enable this.

//CC @IG-Stanimir-Dimitrov